07-MemoryManagement/module/x86_64_out.txt

```diff
@@ -0,0 +1,166 @@
+size/dt     kmalloc     kfree       vmalloc     vfree       kzmalloc    kzfree      getFreePage freePages   
+2           360         155         54461       1111        154         258         211         88          
+2           540         280         71965       1845        360         412         330         180         
+2           541         244         61076       1584        161         195         271         118         
+2           542         210         43356       1664        367         365         305         192         
+2           605         295         79718       2405        308         465         350         217         
+4           121         47          554         354         71          72          79          47          
+4           130         128         1275        740         142         158         158         102         
+4           170         127         1045        391         60          75          78          47          
+4           180         110         1313        1075        135         180         157         102         
+4           200         115         1348        855         208         177         160         102         
+8           158         130         925         632         192         227         160         102         
+8           158         193         1025        427         190         193         157         98          
+8           293         122         488         319         90          98          78          56          
+8           295         162         960         900         197         167         160         102         
+8           78          69          684         324         92          86          79          46          
+16          1237        772         48181       3360        10438       9244        158         97          
+16          1290        812         76243       3395        13413       9227        158         100         
+16          1416        939         61797       1965        6689        4573        79          47          
+16          1545        760         93206       3690        12790       9187        157         100         
+16          755         404         58259       1643        5481        4679        79          55          
+32          0           0           0           0           0           0           157         115         
```

Comment:
Is the size of memory allocated by get_free_pages greater than size of vmalloc?

Reply:
Fixed bug. I set the size of the allocated memory incorrectly.